src/hooks/useBankTransactions/useBankTransactions.tsx

```diff
@@ -347,10 +347,7 @@ export const useBankTransactions: UseBankTransactions = params => {
   const shouldHideAfterCategorize = (
     bankTransaction: BankTransaction,
   ): boolean => {
-    return (
-      filters?.categorizationStatus === DisplayState.review &&
-      ReviewCategories.includes(bankTransaction.categorization_status)
-    )
```

Comment:
This seems probably fine, but just checking why this is? Is it hiding matches?

Reply:
This was being called *after* a transaction was categorized/matched, so the `ReviewCategories.includes(bankTransaction.categorization_status)` was never true